README.md

```diff
@@ -24,28 +24,30 @@ COM has been superseded by [WinRT](https://docs.microsoft.com/en-us/windows/uwp/
 To both consume or produce a COM component through an interface, you will first need to generate the Rust representation of said interface. The `com_interface` macro is the main tool for automatically generating this Rust representation.
 
 ```rust
-#[com_interface("00000000-0000-0000-C000-000000000046")]
-pub trait IUnknown {
-    unsafe fn query_interface(
-        &self,
-        riid: *const IID,
-        ppv: *mut *mut c_void
-    ) -> HRESULT;
-    fn add_ref(&self) -> u32;
-    unsafe fn release(&self) -> u32;
-}
+com::com_interface! {
```

Comment:
Not sure about the name of `com_interface` now that we can define multiple interfaces and that's likely to the norm. Also, the redundant `com`. Perhaps something like `com::declare` e.g. [declaration](https://en.wikipedia.org/wiki/Declaration_(computer_programming)).

Reply:
For now I've gone with `com::interfaces!`. I like `declare!`, but this only really works if we combine the interfaces and classes declarations together which is possible, but not trivial to implement since it's sometimes hard to know in parsing if you're dealing with an interface or a class.